atest/testdata/standard_libraries/collections/dictionaries_should_be_equal.robot

```diff
@@ -134,3 +140,15 @@ Different values and custom error message with values
     ...    Dictionary {'a': 1, 'A': 2} contains multiple keys that are normalized to 'a'. \
     ...    Try normalizing only dictionary values like 'ignore_case=values'.
     Dictionaries Should Be Equal    ${{{'a': 1, 'A': 2}}}    ${{{'a': 2, 'A': 1}}}    ignore_case=True
+
+`ignore_value_order` set to True
+    Dictionaries Should Be Equal    ${D10A}    ${D10B}    ignore_value_order=True
+    
+`ignore_value_order` set to True when dictionaries has key with value of a list of a dictionaries
+    Dictionaries Should Be Equal    ${D10C}    ${D10D}    ignore_value_order=True
```

Comment:
Instead of using variables created in the Variables section just for these tests, they could inline Python expressions similarly as in the test above. I believe most of the existing variables predate the inline expression syntax, and in cases where variable is reused creating it in Variables is a good idea as well.

The above said, this approach is fine as well. You can leave it like it is if you prefer it that way.

Reply:
leaving this as is